tests/js/client/server_parameters/test-telemetrics-true-auth.js

```diff
@@ -0,0 +1,777 @@
+
+
+        db._createDatabase(databaseName);
+        db._useDatabase(databaseName);
+
+        let numShards = 1;
+        if (isCluster) {
+          numShards = 2;
+        }
+        db._create(cn, {numberOfShards: numShards, replicationFactor: 1});
+        for (let i = 0; i < 2000; ++i) {
+          db[cn].insert({value: i + 1, name: "abc"});
+        }
+        db[cn].ensureIndex({type: "persistent", fields: ["value"], name: "persistentIdx1"});
+        db[cn].ensureIndex({type: "geo", geoJson: true, fields: ["geo"], name: "geoIdx1"});
+        db[cn].ensureIndex({type: "geo", geoJson: true, fields: ["otherGeo"], name: "geoIdx2"});
+
+        db._createView('testView1', 'arangosearch', {});
+        db._useDatabase("_system");
+
+        arango.restartTelemetrics();
+
+        telemetrics = getTelemetricsResult();
+        assertForTelemetricsResponse(telemetrics);
+        deployment = telemetrics["deployment"];
+        assertEqual(deployment["databases"].length, 2);
+        const databases = deployment["databases"];
+        const db1Idx = databases[0]["n_views"] === 1 ? 0 : 1;
+        assertEqual(databases[db1Idx]["n_views"], 1);
+        let numColls = 0;
+        databases.forEach((database, idx) => {
+          let totalNumDocs = 0;
+          if (idx === db1Idx) {
+            // there are already the 8 system collections in each database + 1 created here
+            assertEqual(database["n_doc_colls"], 10);
+            database["colls"].forEach(coll => {
+              const nDocs = coll["n_docs"];
+              totalNumDocs += nDocs;
+              if (nDocs === 2000) {
+                assertEqual(coll["n_shards"], numShards);
+                assertEqual(coll["rep_factor"], 1);
+                assertEqual(coll["n_primary"], 1);
+                assertEqual(coll["n_persistent"], 1);
+                assertEqual(coll["n_geo"], 2);
+                numColls++;
+                assertEqual(coll.idxs.length, 4);
+              } else {
+                assertEqual(nDocs, 0);
+                //system collections have replication factor 2
+                if (!isCluster) {
+                  assertEqual(coll["rep_factor"], 1);
+                }
+                assertEqual(coll["n_primary"], 1);
+                assertEqual(coll["n_persistent"], 0);
+                assertEqual(coll["n_geo"], 0);
+              }
+            });
+            assertEqual(totalNumDocs, 2000);
+          } else {
+            //includes the collections created for the smart graph
+            assertEqual(database["n_doc_colls"], 15);
+            database["colls"].forEach(coll => {
+              assertEqual(coll["n_primary"], 1);
+              assertEqual(coll["n_persistent"], 0);
+              if (!isCluster) {
+                assertEqual(coll["rep_factor"], 1);
+              }
+              assertEqual(coll["n_geo"], 0);
+            });
+          }
+        });
+      } finally {
+        try {
```

Comment:
There should be a `db._useDatabase("_system");` in the finally part.
It currently is somewhere in the try part, but it is not guaranteed that it was reached, so the finally may run and try to drop the database while not being in the `_system` database itself. Then the `_dropDatabase` in the finally will throw.